okhttp/src/main/java/okhttp3/Handshake.java

```diff
@@ -46,13 +47,14 @@ private Handshake(TlsVersion tlsVersion, CipherSuite cipherSuite,
     this.localCertificates = localCertificates;
   }
 
-  public static Handshake get(SSLSession session) {
+  public static Handshake get(SSLSession session) throws IOException {
```

Comment:
Adding a checked exception isn’t source-compatible. I’m okay with that but we’ll need to call it out in the changelog.